en/components/select.md

```diff
@@ -34,20 +34,71 @@ import { IgxSelectModule } from 'igniteui-angular';
+</igx-select>
+```
+
+Another way to do it would be to use a collection of the items that we want to display with the [ngForOf](https://angular.io/api/common/NgForOf) directive:
+
+```typescript
 public items: string[] = ["Orange", "Apple", "Banana", "Mango"];
 ```
 
-Then in your template you need to bind it with said items like so:
 ```html
-<igx-select>
+<igx-select [(ngModel)]="selected">
+    <igx-select-item *ngFor="let item of items" [value]="item">
+        {{item}}
+    </igx-select-item>
+</igx-select>
+```
+Additionally, the above sample illustrates two-way data-binding via `ngModel`. What is more, you can use the Select component in [`Angular Forms`](#select-in-angular-forms) .
+
+
+
+### Label, Prefix and Suffix
+The Select supports the following directives applicable to the [Input Group](input_group.md):
+
+- `igxLabel` - No need to set the `for` property, as linking with the Select input is handled automatically via `aria-labelledby`.
+- `igx-prefix`/`igxPrefix`
+- `igx-suffix`/`igxSuffix` - Note the built-in toggle button suffix will always be displayed last.
+
+```html
+<igx-select [(ngModel)]="selected">
+    <label igxLabel>Pick a fruit</label>
+    <igx-prefix>
+        <igx-icon color="green" style="width:48px; height:32px;">BIO</igx-icon>
+    </igx-prefix>
+    <igx-suffix *ngIf="selected">
+        <igx-icon (click)="clearSelection($event)">clear</igx-icon>
+    </igx-suffix>
     <igx-select-item *ngFor="let item of items" [value]="item">
         {{item}}
     </igx-select-item>
 </igx-select>
 ```
-Notice that we use an `IgxSelectItemComponent` to display the items that `igx-select` operates on. The `IgxSelectItemComponent` extends the [*IgxDropDownItemComponent*](https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/classes/igxdropdownitemcomponent.html) with additional functionality that is specific to the `igx-select-item`.
+
+<div class="sample-container loading" style="height: 350px;">
+    <iframe id="select-input-directives-iframe" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/data-entries/select-input-directives" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="select-input-directives-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+
+> [!NOTE]
+If no [`placeholder`]({environment:angularApiUrl}/classes/igxselectcomponent.html#placeholder) is specified for the Select component and there is no selection made, the `igxLabel` will transition and appear where you would expect the placeholder to be.
+
+### Templating toggle button
+The default arrow can be replaced using a template with the help of [`toggleIconTemplate`]({environment:angularApiUrl}/classes/igxselectcomponent.html#toggleicontemplate) directive. For example:
+```html
+<ng-template igxSelectToggleIcon let-collapsed>
```

Comment:
Should be clear the template w/ directive should be inside the component definition.
```suggestion
<igx-select #select>
  ...
  <ng-template igxSelectToggleIcon let-collapsed>
```